arangod/Transaction/Methods.cpp

```diff
@@ -1970,8 +1973,8 @@ DataSourceId transaction::Methods::addCollectionAtRuntime(
           TRI_ERROR_TRANSACTION_UNREGISTERED_COLLECTION,
           std::string(
               TRI_errno_string(TRI_ERROR_TRANSACTION_UNREGISTERED_COLLECTION)) +
-              ": " + collectionName + " [" + AccessMode::typeString(type) +
-              "]");
+              ": " + std::string{collectionName} + " [" +
```

Comment:
consider using `absl::StrCat(...)` here.

Reply:
Added in: efe727464c8e5114c22a6fc2a3433ece50ac9527